packages/babel-config/src/plugins/babel-plugin-redwood-directory-named-import.ts

```diff
@@ -42,7 +42,7 @@ export default function ({ types: t }: { types: typeof types }): PluginObj {
           // CONTINUE...
         }
 
-        const newPath = getNewPath(value, <string>filename)
+        const newPath = getNewPath(value, filename as string)
```

Comment:
https://typescript-eslint.io/rules/consistent-type-assertions: `<...>...` type assertions are somewhat legacy, and can't be used in `.tsx` files because they're ambiguous with some JSX syntax. `... as ...` is generally preferred.